boards/DCM/Src/App/states/App_SharedStates.c

```diff
@@ -31,3 +31,49 @@ void App_SharedStatesRunOnTick100Hz(struct StateMachine *const state_machine)
     App_CanTx_SetPeriodicSignal_LEFT_INVERTER_SWITCH(
         can_tx, App_InverterSwitches_IsLeftOn(inverter_switches));
 }
+
+void App_SharedStates_ConfigInverterSwitches(struct DcmWorld *world)
+{
+    struct DcmCanRxInterface *can_rx = App_DcmWorld_GetCanRx(world);
+    struct InverterSwitches * inv_switches =
+        App_DcmWorld_GetInverterSwitches(world);
+
+    // Close or open the inverter LV switches if requested by a PCAN node
```

Comment:
with the changes suggested in the dbc, this can be 2 if-else statements

Reply:
done